### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: src/AppBundle/Controller/MainController.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ class MainController extends Controller
      * @Route("/", name="app_main_index")
      * @Method("GET")
      */
-    public function indexAction(Request $request): Response
+    public function indexAction(): Response
     {
         return $this->render('main/index.html.twig');
     }
```
Comment (Dam2a) [inline src/AppBundle/Controller/MainController.php]: This change does not belong to this PR.

Comment (HeahDude) [reply src/AppBundle/Controller/MainController.php]: It could since we test this method, we're allowed to fix ;).

Comment (Dam2a) [reply src/AppBundle/Controller/MainController.php]: OK. It's good.
